go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/chanzuckerberg/fogg
 
-go 1.19
+go 1.21
```

Comment:
this will break dependabot, golang 1.21 changed the version format:

```
Dependabot encountered the following error:

go: downloading go1.21 (linux/amd64)
go: download go1.21 for linux/amd64: toolchain not available
```

https://github.com/chanzuckerberg/fogg/pull/856

also, after upgrading locally, may need to upgrade vscode golang tooling:
https://github.com/golang/go/issues/61888#issuecomment-1716987059